app/controllers/posts_controller.rb

```diff
@@ -1,6 +1,7 @@
 class PostsController < ApplicationController
   def index
     @subcategories = MarkdownDocument.all_subcategories(params[:section])
+    not_found if params[:section] == "transcripts"
```

Comment:
should this not trigger earlier to avoid calling (the presumably invalid) MarkdownDocument.all_subcategories("transcripts") ?

Reply:
Sure, can do!